doc/attributes.md

```diff
@@ -0,0 +1,51 @@
+# User And Group Attributes
+
+This document describes the attributes that can be indicated for users and groups, either through the YaST UI of the *users* client or through AutoYaST. Each attribute should have a counterpart in the new *Y2Users* code. This correspondence helps to translate the users data structure from *Yast::Users* perl module to the new code.
+
+## User
+
+The YaST UI for creating and editing a user offers the following attributes:
+
+| UI Field | Y2Users attr | Only when adding | Only when editing | Description |
+| :--- | :--- | :--- | :--- | :--- |
+| full name | gecos | | | |
+| name | name | | | |
+| password | password | | | |
+| system mail | | | | |
+| disable login | * password starting by !$ | | | |
+| uid | uid | | | |
+| home dir | home | | | | |
+| home dir permission | | | | |
+| empty home | | yes | | |
+| Move to new location | | | yes |  |
+| Btrfs subvolume | btrfs_subvolume_home | | | |
+| Additional info | gecos | | | |
+| login shell | shell | | | |
+| group | primary_group | | | |
+| additional groups | groups | | | |
+
+### Home management
+
+### Creating a user
+
+* A user can be created with or without home.
+* To create a user without home, the *home dir* path should be set to empty.
+* The home can be created with or without default content (*empty home* checkbox).
+* The home can be created as Btrfs subvolume (*Btrfs subvol* checkbox).
+
+### Editing a user
+
+* If the home path is changed, then a new home is created with the default content.
+* There is no way to create a new home without content (*empty home* checkbox is not available).
+* The old home is removed.
```

Comment:
From the top of my head, I believe that not always, Only when moving the content to a new location